### ISSUE TITLE
iOS emulators are not displayed in the iOS list

### ISSUE THREAD

**Issue by 381181295** [sudharsan-selvaraj/emulator-manager#1]
The iOS emulators are not showing up inside VSCode. (Even though Xcode and the command line tools are installed and updated)
![image](https://user-images.githubusercontent.com/78950093/229615430-74878a7e-6ca2-4250-b22e-efa3358f5e40.png)

Here are my system specs:
- Chip: Apple M1 Pro
- macOS: Ventura 13.3
- Xcode: 14.3 (14E222b)
- Xcode Command Line Tools: 14.0.3 (clang-1403.0.22.14.1)
  - Target: arm64-apple-darwin22.4.0
  - Thread model: posix
  - InstalledDir: /Library/Developer/CommandLineTools/usr/bin

Android emulators are working as expected.

Let me know if i'm missing any details that may help pin down the issue.

**Comment by sudharsan-selvaraj** [comment]
Can you run the below comand in terminal and share me what's the output looks like

`xcrun simctl `

**Comment by 381181295** [comment]
Ah nice catch, i was getting this error:
`xcrun: error: unable to find utility "simctl", not a developer tool or in PATH`

Fixed that by going to Xcode > Settings > Locations. The `Command Line Tools` was already set to `Xcode 14.3 ...` but the file path below it was not working for some reason. So I just reselected `Command Line Tools = Xcode 14.3 ...`, it prompted me for my password, then the path changed to `/Application/Xcode.app`  and after that all my iOS emulators were showing up in Simcode.

Thanks!!
